lib/pbench/test/unit/server/test_datasets_list.py

```diff
@@ -345,6 +345,43 @@ def test_get_bad_keys(self, query_as):
             )
         }
 
+    def test_get_get_errors(self, server_config, query_as):
+        """Test case reporting key errors
+
+        Args:
+            query_as: Query helper fixture
+        """
+        fio_1 = Dataset.query(name="fio_1")
+        fio_2 = Dataset.query(name="fio_2")
+        Metadata.setvalue(dataset=fio_1, key="global.test", value="ABC")
+        Metadata.setvalue(dataset=fio_2, key="global.test.foo", value="ABC")
```

Comment:
If you do another edit pass, it seems like it would be good to have this sort of overlap in `test_key_summary()`.